CMakeLists.txt

```diff
@@ -764,10 +768,14 @@ if (BUILD_TESTS)
       TEST dynamic_test SOURCES DynamicTest.cpp
       TEST dynamic_converter_test SOURCES DynamicConverterTest.cpp
       TEST dynamic_other_test SOURCES DynamicOtherTest.cpp
+      TEST enable_shared_from_this SOURCES EnableSharedFromThisTest.cpp
       TEST endian_test SOURCES EndianTest.cpp
+      TEST exception_string_test BROKEN SOURCES ExceptionStringTest.cpp
       TEST exception_test SOURCES ExceptionTest.cpp
       TEST exception_wrapper_test SOURCES ExceptionWrapperTest.cpp
+      TEST expected_coroutines_test SOURCES ExpectedCoroutinesTest.cpp
       TEST expected_test SOURCES ExpectedTest.cpp
+      TEST fbstring_test SOURCES FBStringTest.cpp
```

Comment:
Note: `fbstring` had a problem building with msvc, fixed (and also enabling in cmake) in 8c458a4d6d2b09bd95556505f68278eb93242c11. So this line will be rebased out.

Reply:
Yep, I saw 8c458a4 go in today -- nice fix 👍 

Rebased with top of tree.

Reply:
@yfeldblum when your `fbstring` fix landed earlier, it accidentally broke CMake 3.9+ users as discussed in this PR. I've made a separate PR with that test comment fix -> https://github.com/facebook/folly/pull/1080